modules/campaignion_email_protest_type/campaignion_email_protest_type.install

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @file
+ * Update hooks for the email protest feature.
+ */
+
+/**
+ * Delete node type.
+ */
+function campaignion_email_protest_type_update_4() {
+  node_type_delete('email_protest');
+}
+
+/**
+ * Remove default template node.
+ */
+function campaignion_email_protest_type_update_3() {
+  foreach (entity_get_id_by_uuid('node', ['7f2e3be8-156e-4211-a35a-a654ff4ab99e']) as $nid) {
+    node_delete($nid);
+  }
```

Comment:
I'm not sure it’s always safe to delete the default template node. It could also be used as a live action and might contain submissions. Why not keep it?

Reply:
Good point. Then lets just not do it.